deploy/example/storageclass-blobfuse2.yaml

```diff
@@ -12,6 +12,7 @@ volumeBindingMode: Immediate
 allowVolumeExpansion: true
 mountOptions:
   - -o allow_other
+  - -o direct_io
```

Comment:
pls remove the new mount options, I have already fixed the external test failure by: https://github.com/kubernetes-sigs/blob-csi-driver/pull/1029

Reply:
done